template.hbr

```diff
@@ -289,6 +300,11 @@ players:
                     "font_style": null,
                     "font_weight": null
                 },
+                "string.special.symbol": {
+                    "color": "#{{rosewater}}",
+                    "font_style": "italic",
+                    "font_weight": null
+                },
```

Comment:
We've moved away from rosewater in the newer style guide too, I'm curious how VSCode highlights these?

Reply:
We had some discussion about this on the discord, will update the style guide soon but this should now be `peach`

The reasoning being that these symbols are usually immutable/constants in languages

```suggestion
                "string.special.symbol": {
                    "color": "#{{peach}}",
                    "font_style": "italic",
                    "font_weight": null
                },
```